changelog/123.feature.rst

```diff
@@ -0,0 +1 @@
+Make the imaging demo compatible with :class:`~xrayvision.visibility.Visibilities`
```

Comment:
To make this work we need to tell sphinx where to look so I think we can add it here https://github.com/TCDSolar/stixpy/blob/da097314db525b8338c118d97d681539373c3874/docs/conf.py#L78-L84 e.g

```
"xrayvision": ("https://xrayvision.readthedocs.io/en/stable/", None)
``` 
but I don't know if anything else is needed